pyracing/client.py

```diff
@@ -370,14 +412,18 @@ async def event_results(
         }
         url = ct.URL_RESULTS
         response = await self._build_request(url, payload)
-
-        event_result_dict = response.json()['d']
-        if event_result_dict:
-            return [historical_data.EventResults(x)
-                    for x in response.json()["d"]["r"]]
-        else:
+        
+        if not response.json():
             return []
-
+        
+        mapping = response.json().get('m')
+        response_items = response.json().get('d').get('r')
+        
+        if mapping and response_items:
+            renamed_items = [self._rename_numerical_keys(ri, mapping) for ri in response_items]
+            
+            return [historical_data.EventResults(x) for x in renamed_items]
```

Comment:
Same here, we should have some return value

Reply:
See above